docs/raw/push-notification-server.md

```diff
@@ -110,14 +109,13 @@ The content of the message MUST contain the following [protobuf record](https://
 
 ```protobuf
 
-message PushNotificationOptions {
+message PushNotificationFilterSettings {
   boolean enabled = 1;
   repeated string allowed_user_list = 2;
-  repeated string block_user_list = 3;
-  repeated string block_chat_list = 4;
+  repeated string blocked_chat_list = 3;
 }
 
-message PushNotificationDeviceToken {
+message PushNotificationOptions {
```

Comment:
Sorry, I might have earlier suggested renaming "DeviceToken" here but that seems to be more appropriate than "Options". It may be a bit confusing because we have this `token` along with `access_token` now.